mentors/requirements.txt

```diff
@@ -0,0 +1,14 @@
+certifi==2019.6.16
+chardet==3.0.4
+ffmpy==0.2.2
+gdown==3.8.1
+idna==2.8
+numpy==1.16.4
+pandas==0.24.2
+python-dateutil==2.8.0
+pytz==2019.1
+requests==2.22.0
+six==1.12.0
```

Comment:
curious, is `six` here for a python 2 dependency in some other package we use or why?

Reply:
I think that got brought in with a different package. I can remove it and see if anything breaks

Reply:
No issues removing it. I'm not sure why it was there in the first place. Thanks for spotting it.